tests/sklearn/test_sklearn_models.py

```diff
@@ -1539,7 +1551,12 @@ def test_fitted_compiled_error_raises(
     verbose=True,
 ):
     """Test Fit and Compile error raises."""
-    n_bits = min(N_BITS_REGULAR_BUILDS)
+    if get_model_name(model_class) == "KNeighborsClassifier":
```

Comment:
I added a few more of these as I believe they could be come future flaky tests (since `min(N_BITS_REGULAR_BUILDS)` = 6 but we have issues with KNN > 5 bits) cc @kcelia 

also, I made this like this as I'm not sure if the issue is temporary, but if it's not then mabe we just want to replace `min(N_BITS_REGULAR_BUILDS)` by `min(N_BITS_REGULAR_BUILDS + N_BITS_WEEKLY_BUILDS)`